System_Information.bat

```diff
@@ -0,0 +1,29 @@
+@echo off
+
+:: Create directory "System Information" if doesn't exist
+echo Create folder: System Information
+IF NOT EXIST "System Information" md "System Information"
+
+SET filePath="System Information\System_Information.txt"
+
+:: Run SYSTEMINFO
+echo Running command: systeminfo
+echo ------- Output of command: systeminfo --------- > %filePath%
+systeminfo >> %filePath%
+
+:: Run WMIC
+echo Running command: WMIC
+echo ------- Output of command: WMIC --------- >> %filePath%
+wmic /APPEND:%filePath% product get name,vendor,version >> nul
+
+::Run Dxdiag and append data to System_Information.txt
+echo Running command: dxdiag
+echo ------- Output of command: dxdiag --------- >> %filePath%
+dxdiag /T dxdiag.txt
+type dxdiag.txt >> %filePath%
+del dxdiag.txt
+
+::Run PsInfo and append data to System_Information.txt

```

Comment:
Okay, this is the information that PsInfo gives us, ie:

<pre>
System information for \\VAGRANT-10:
Uptime:                    0 days 0 hours 1 minute 19 seconds
Kernel version:            Windows 10 Enterprise Evaluation, Multiprocessor Free
Product type:              Professional
Product version:           6.3
Service pack:              0
Kernel build number:       10240
Registered organization:   Vagrant Inc.
Registered owner:          Vagrant Administrator
IE version:                9.0000
System root:               C:\Windows
Processors:                2
Processor speed:           2.2 GHz
Processor type:            Intel(R) Core(TM) i5-4200U CPU @
Physical memory:           2 MB
Video driver:              VirtualBox Graphics Adapter for Windows 8+
</pre>

I'd say that, given that systeminfo and dxdiag are already providing some of this information, we can just get rid of it  but still, PsInfo gives us:
* product type
* product version
* service pack
* kernel build number
* ie version
* system root - but systeminfo gives us the location of some paths such as windows or system folders

Honestly, I'm not really sure whether all these are strictly relevant to us given the amount of information that we're already collecting from the system. My vote goes for removing PsInfo and retrieving the information that we really need from elsewhere (and that comes out of the box). @stegru and @kavyakomuru?